### PR TITLE
[zh] Resync docs/concepts/storage/volumes.md

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -1755,34 +1745,103 @@ spec:
...
+for production use.
+
+The PHP application's code and assets map to the volume's `html` folder and
+the MySQL database is stored in the volume's `mysql` folder. For example:
 -->
-下面是一个使用同一共享卷的、内含 LAMP 栈（Linux Apache Mysql PHP）的 Pod 的示例。
-HTML 内容被映射到卷的 `html` 文件夹，数据库将被存储在卷的 `mysql` 文件夹中：
+下面例子展示了如何配置某包含 LAMP 堆栈（Linux Apache MySQL PHP）的 Pod 使用同一共享卷。
+此示例中的 `subPath` 配置不建议在生产环境中使用。
+PHP 应用的代码和相关书局映射到卷的 `html` 文件夹，MySQL 数据库存储在卷的 `mysql` 文件夹中：
```
Comment (zhiguo-lu) [inline content/zh/docs/concepts/storage/volumes.md:1844]: 代码和相关书局 -> 代码和网站资源

File: content/zh/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -343,55 +345,55 @@ spec:
...
+entry in the ConfigMap. The following configuration shows how to mount
+the `log-config` ConfigMap onto a Pod called `configmap-pod`:
 -->
-
-当引用 `configMap` 对象时，你可以简单的在 Volume 中通过它名称来引用。
-还可以自定义 ConfigMap 中特定条目所要使用的路径。
-例如，要将名为 `log-config` 的 ConfigMap 挂载到名为 `configmap-pod` 的 Pod 中，您可以使用下面的 YAML：
+引用 configMap 对象时，你可以在 volume 中通过它的名称来引用。
+你可以自定义 ConfigMap 中特定条目所要使用的路径。
+下面的配置羡慕吃如何将名为 `log-config` 的 ConfigMap 挂载到名为 `configmap-pod`
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/storage/volumes.md:395]: Done.

File: content/zh/docs/concepts/storage/volumes.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/concepts/storage/volumes.md:1844]: Done

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**